src/pages/trading-specification/components/_elements.tsx

```diff
@@ -236,12 +235,18 @@ export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const { symbol, instrument: text, dl_icon, swf_icon } = data
     const [show_popUp, setShowPopUp] = useState(false)
     const [popup_type, setPopupType] = useState<TPopupType>()
+    const [scrollY, setScrollY] = useState<number>()
```

Comment:
```suggestion
    const [scroll_y, setScrollY] = useState<number>()
```

Reply:
According to our documentation we just let the types infer in this case `number` type is not required to be added and instead can we make the value `0` as default?